tests/StrictObjects/SuggesterTest.phpt

```diff
@@ -55,14 +57,20 @@ class SuggesterTest extends \Tester\TestCase
 	}
 
 	/**
+	 * @param string|null $expected
```

Comment:
This is redundant, but `Squiz.Commenting.FunctionComment.ParamNameNoMatch` sniff is broken and requires it, otherwise it complains about _Doc comment for parameter $items does not match actual variable name $expected_. :/